docs/UserGuide.md

```diff
@@ -326,32 +329,35 @@ Example:
 
 ### **Profile**
 
-Note: You must be on the **Profile** tab in order to execute these commands.
+The profile feature offers you the ability to keep track and maintain your personal portfolio. It stores
+profile items which are entities that describe an achievement, a past experience or a skill you have acquired.
+It assists you in crafting a tailored resume for your internship applications by allowing you to find items in your
+profile which are relevant to the job description through the [find feature](#finding-items-in-profile-find-me).
 
-Your profile can contain 3 categories of information.
+Each profile item belongs to  1 of 3 categories.
 
 Valid `CATEGORY` specifiers:
 - `achievement`
 - `experience`
 - `skill`
 
-Each profile item (bit of information) also contains a `TITLE` that describes the item, and you can optionally 
+Each profile item also contains a `TITLE` that describes the item, and you can optionally 
 add additional `DESCRIPTOR`s to further describe the item in point form.
 
 #### Adding item to profile: `add me`
 
-Adds experience, skills or achievements descriptors to your profile.
+Adds a profile item to your profile.
 
 Format: `add me c/CATEGORY t/TITLE [d/DESCRIPTOR]...`
 
-* Category specifies a category which can be either experience, skills or achievement.
+* `CATEGORY` specifies a category which can be either experience, skills or achievement.
```

Comment:
not sure if this is a repetition of line 333 to 336. If it isn't you might want to encase experience skills achievements with the `experience` individually

Reply:
Line 333 is a brief description of profile items and 336 is a more formal definition. I think its mainly for the overview in  2101.